Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCollectionOperations.swift

```diff
@@ -640,17 +640,38 @@ struct UnifiedDistinct: UnifiedOperationProtocol {
     /// Filter for the query.
     let filter: BSONDocument
 
+    /// Options to use for the operation.
+    let options: DistinctOptions?
+
     /// Optional identifier for a session entity to use.
     let session: String?
 
     static var knownArguments: Set<String> {
-        ["fieldName", "filter", "session"]
+        ["comment", "fieldName", "filter", "session"]
+    }
+
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case fieldName, filter, options, session
```

Comment:
`options` isn't actually a coding key, since all of the options arguments are contained at the same level. so we can drop that from this enum

Reply:
dropped